pypeit/scripts/chk_edges.py

```diff
@@ -17,26 +17,28 @@ def parser(options=None):
     parser.add_argument('root', type=str, default = None, help='PYPIT Master Trace file root [e.g. MasterTrace_A_01_aa.fits]')
     parser.add_argument("--chname", default='MTrace', type=str, help="Channel name for image in Ginga")
     parser.add_argument("--dumb_ids", default=False, action="store_true", help="Slit ID just by order?")
+    parser.add_argument("--old", default=False, action="store_true", help="Used old slit tracing")
     #parser.add_argument("--show", type=str, help="Use the show() method of TraceSlits to show something else")
 
-    if options is None:
-        args = parser.parse_args()
-    else:
-        args = parser.parse_args(options)
-    return args
+    return parser.parse_args() if options is None else parser.parse_args(options)
 
 
 def main(pargs):
 
-    import pdb as debugger
     import time
 
     from pypeit import ginga
     from pypeit import traceslits
     from pypeit.core.trace_slits import get_slitid
+    from pypeit import edgetrace
 
     import subprocess
 
+    if not pargs.old:
+        edges = edgetrace.EdgeTraceSet.from_file(pargs.root)
+        edges.show(thin=10, in_ginga=True)
```

Comment:
Maybe make the ginga bit optional so that people without a functional
ginga can still use this.